docker-for-mac/release-notes.md

```diff
@@ -13,6 +13,34 @@ This page contains information about the new features, improvements, known issue
 
 {% include eula.md %}
 
+## Docker Desktop 3.6.0
+2021-08-11
+
+[Download for Mac with Intel chip](https://desktop.docker.com/mac/stable/amd64/66501/Docker.dmg)
+[Download for Mac with Apple chip](https://desktop.docker.com/mac/stable/arm64/66501/Docker.dmg)
```

Comment:
I think the latest release should have `{: .button .primary-btn}` on these?

Reply:
Shouldn't this be to point to the stable release? 

```
[Mac with Intel chip](https://desktop.docker.com/mac/stable/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn}
[Mac with Apple chip](https://desktop.docker.com/mac/stable/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){: .button .primary-btn}
```

Reply:
This change should be on the Docker Desktop 3.5.2 release notes

https://github.com/docker/docker.github.io/blob/7998bfa0750c96108927eac82ae3296cc9294685/docker-for-mac/release-notes.md#docker-desktop-352